src/App.scss

```diff
@@ -64,11 +65,6 @@
   }
 
   .drift-toolbar {
-    padding: 20px;
-    padding-top: 0px;
-  }
-
-  .drift-modal-toolbar {
     padding: 16px 32px;
```

Comment:
This is throwing off the padding for the toolbar in the System Comparison table. You can compare it's state in this PR to ci and see the difference.